charms/sackd/src/charm.py

```diff
@@ -99,13 +102,15 @@ def _on_slurmctld_available(self, event: SlurmctldAvailableEvent) -> None:
 
         # Restart sackd after we write event data to respective locations.
         self._sackd.munge.service.restart()  # TODO change this once auth/slurm in place
-        if self._sackd.service.active():
-            self._sackd.service.restart()
-        else:
-            self._sackd.service.start()
+        try:
+            if self._sackd.service.active():
+                self._sackd.service.restart()
+            else:
+                self._sackd.service.start()
+        except SlurmOpsError as e:
+            logger.debug(e)
```

Comment:
[nit]: This should be an error level message:

```suggestion
            logger.error(e)
```